gradle.properties

```diff
@@ -22,6 +22,7 @@ kotlin.code.style=official
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
 android.enableJetifier=false
+android.injected.androidTest.leaveApksInstalledAfterRun=true
```

Comment:
this allows us to collect screenshots off devices after instrumented tests (androidTest) finish